cypress/e2e/inspectlet.spec.cy.ts

```diff
@@ -0,0 +1,28 @@
+describe(
+  'To test Inspectlet integration',
+  () => {
+    beforeEach(() => {
+      cy.visit('/');
+    });
+
+    it('Should enable integration if key was provided', ()=>{
+      cy.window().then(win => {
+        let inspectlet_key = win.inspectlet_key;
+        cy.visit('/?inspectlet_diagnostics=true');
+        if (inspectlet_key) {
+          cy.get('.inspectlet_diagnostics').should('be.visible');
+        }
+      })
+    })
+    
+    it('Should not enable integration if key was not provided', ()=>{
+      cy.window().then(win => {
+        let inspectlet_key = win.inspectlet_key;
+        cy.visit('/?inspectlet_diagnostics=true');
+        if (!inspectlet_key) {
+          cy.get('.inspectlet_diagnostics').should('not.exist');
+        }
+      })
```

Comment:
I'm no sure what's happening here but this test case is definitely not setup and written according to what's required.

Please rewrite the this test case, where:
- You are making sure inspectlet_key is not defined.
- You are making sure `.inspectlet_diagnostics` does not exists.

There should be no need for any conditional expression here.

Reply:
Basically what we are doing is passing inspectlet.wid to client side and storing that in window object, and if it's not passed we are initialising window.inspectlet_key to empty string...
- `/?inspectlet_diagnostics=true` Using this inspectlet support, inspectlet creates a div inside our page with a classname `.inspectlet_diagnostics`
- I used conditional statements because that was the only option to determine whether inspectlet_key is empty or not (or we can say inspectlet_key is passed or not)...
- If we will have an empty string in the window.inspectlet_key, inspectlet code will not run and it will not create `.inspectlet_diagnostics` div, if we will have a string in window.inspectlet_key inspectlet code will be injected and it will create `.inspectlet_diagnostics` div.... So I am using this to test actions according to inspectlet.wid value

Reply:
Please find a way to make test cases work without conditional expressions